### PR TITLE
Task/frontend78 main video card

### PR DESCRIPTION
**Pull request by Barracuda713**
This PR contains:
- add Videa Card for main page
- fix redirect on auth pages
- fix auth pages styles

### CODE REVIEW

File: frontend/src/components/common/video-card-main/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  height: 100%;
+}
+
+.video-card-play-btn {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  background-color: #f41b3b;
```
Comment (droztoff) [inline frontend/src/components/common/video-card-main/styles.module.scss]: use css variables

File: frontend/src/components/common/video-card-main/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+  z-index: 15;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  gap: 10px;
+  padding: 6px 15px;
+  color: var(--white-text-color);
+  font-size: 10px;
+  line-height: 14px;
+  background-color: rgb(0 0 0 / 70%);
```
Comment (droztoff) [inline frontend/src/components/common/video-card-main/styles.module.scss]: css variables

**Review by droztoff — DISMISSED** [review]

**Review by droztoff — DISMISSED** [review]

**Comment by Barracuda713** [comment]
[Unable to return to the main page from the authorization pages](https://trello.com/c/2cVj1jDh/102-unable-to-return-to-the-main-page-from-the-authorization-pages)

**Comment by Barracuda713** [comment]
[Sign-in/Sign-up/Restore password pages text issues](https://trello.com/c/SfzwcTXV/81-sign-in-sign-up-restore-password-pages-text-issues)

**Comment by Barracuda713** [comment]
[The "Sign out" button in the header redirects user to the Sign-in page after click](https://trello.com/c/NO3suXCB/104-the-sign-out-button-in-the-header-redirects-user-to-the-sign-in-page-after-click)

**Review by shirokuronemuri — APPROVED** [review]
There are several issues, but since Artem has already merged this branch into his one (#77), I think it's better if you fix this issues in a separate bugfix.
For other reviewers, check #77's "redirected the issue to @Barracuda713" comments

**Comment by Barracuda713** [comment]
@ihor-pavliuk Yes, as we agreed earlier

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**